jans-cedarling/cedarling/src/log/memory_logger.rs

```diff
@@ -37,7 +37,8 @@ impl LogWriter for MemoryLogger {
 
         let result = self
             .storage
-            .borrow_mut()
+            .lock()
+            .unwrap()
```

Comment:
I understand that this may never happen, since the `Mutex` might never be held over an async state, but, please handle the error, at least, use an `expect()`, with an error message so we know where it originated.

Reply:
Done